### PR TITLE
feat: Update asserts to use new winter23 Assert class - Invocable Recipes

### PR DESCRIPTION
**Pull request by msrivastav13**
### What does this PR do?

Update asserts to use new winter23 Assert class. The new Assert class is really elegant compared to the older System methods.

### What issues does this PR fix or reference?

#<Insert GitHub Issue>

## The PR fulfills these requirements:

[ ] Tests for the proposed changes have been added/updated.
[ ] Code linting and formatting was performed.

### Functionality Before

<insert gif and/or summary>

### Functionality After

<insert gif and/or summary>

### CODE REVIEW

File: force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls [hunk 1]
```diff
@@ -7,7 +7,7 @@ private class InvocableMethodRecipes_Tests {
         request.inputs = new List<Account>{ new Account(name = 'test') };
         Test.stopTest();
 
-        System.assertEquals(
+        System.Assert.areEqual(
```
Comment (pozil) [inline force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls:10]: Fix param order

File: force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls [hunk 2]
```diff
@@ -35,8 +35,11 @@ private class InvocableMethodRecipes_Tests {
         );
         Test.stopTest();
 
-        System.assert(results.size() == 1, 'Expected to receive one contact');
-        System.assert(
+        System.Assert.isTrue(
```
Comment (pozil) [inline force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls:38]: Change to `areEqual`

File: force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls [hunk 3]
```diff
@@ -35,8 +35,11 @@ private class InvocableMethodRecipes_Tests {
...
+            results.size() == 1,
+            'Expected to receive one contact'
+        );
+        System.Assert.isTrue(
```
Comment (pozil) [inline force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls:42]: Change to `areEqual`

File: force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls [hunk 4]
```diff
@@ -67,8 +70,11 @@ private class InvocableMethodRecipes_Tests {
         );
         Test.stopTest();
 
-        System.assert(results.size() == 1, 'Expected to receive one contact');
-        System.assert(
+        System.Assert.isTrue(
```
Comment (pozil) [inline force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls:73]: Change to `areEqual`

File: force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls [hunk 5]
```diff
@@ -67,8 +70,11 @@ private class InvocableMethodRecipes_Tests {
...
+            results.size() == 1,
+            'Expected to receive one contact'
+        );
+        System.Assert.isTrue(
```
Comment (pozil) [inline force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls:77]: Change to `areEqual`

**Review by pozil — CHANGES_REQUESTED** [review]
Found some minor issues.

File: force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls (hunk 1 above)
Comment (msrivastav13) [reply force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls:10]: Ah I see now that some of my commits were not pushed up! Thanks for catching it.

File: force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls (hunk 2 above)
Comment (msrivastav13) [reply force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls]: Fixed!

File: force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls (hunk 3 above)
Comment (msrivastav13) [reply force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls]: Fixed!

File: force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls (hunk 5 above)
Comment (msrivastav13) [reply force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls]: Fixed!

**Pull request merged**